lib/pure/collections/sequtils.nim

```diff
@@ -113,11 +131,12 @@ proc distribute*[T](s: seq[T], num: int, spread = true): seq[seq[T]] =
   ##   assert numbers.distribute(6)[0] == @[1, 2]
   ##   assert numbers.distribute(6)[5] == @[7]
   assert(not s.isNil, "`s` can't be nil")
-  assert(num > 0, "`num` has to be greater than zero")
   if num < 2:
     result = @[s]
     return
 
+  let num = cast[int](num)
```

Comment:
I already answered here. Seems that GitHub loses posts for inline comments with force pushes).

It does not work without the cast because of `pos: Positive` in `for x in 0 .. < pos` fails with `0 is not included for Positive`.